src/Flashback-Decompiler/FBDDecompiler.class.st

```diff
@@ -486,6 +486,22 @@ FBDDecompiler >> pushActiveContext [
 	simulatedStack addLast: builder codeThisContext 
 ]
 
+{ #category : #'closure support' }
+FBDDecompiler >> pushCleanClosure: aCompiledBlock [
+
+	| savedSimStack savedSequence |
+
+	savedSimStack := simulatedStack.
+	savedSequence := currentSequence.
+
+	simulatedStack addLast: (self
+		extractFullBlock: aCompiledBlock numCopied: 0).
+
+	simulatedStack := savedSimStack.
+	currentSequence := savedSequence.
```

Comment:
No tests for this?

Reply:
(that is, decompile and recompile the whole image and then run all tests)